aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -597,6 +598,18 @@ Tensor _sparse_csr_mm(const Tensor& mat1, const Tensor& mat2) {
         0.0,
         1.0);
   }
+  if (mat1.layout() == c10::kStrided &&
+      (mat2.layout() == kSparseCsc || mat2.layout() == kSparseBsc)) {
+    // csc.transpose() -> csr or bsc.transpose() -> bsr
```

Comment:
Same as the comment above. `addmm` should work with `mat2.layout == kSparseBsr` which follows from your added lines.